src/components/AppNavigation/CalendarList/CalendarListNew.vue

```diff
@@ -27,17 +27,25 @@
 		:menu-open.sync="isOpen"
 		menu-icon="icon-add"
```

Comment:
This line needs to be removed once https://github.com/nextcloud/nextcloud-vue/pull/2217 is merged and available here. Otherwise we will see two plus icons (although no one will notice, since they perfectly overlap).